README.adoc

```diff
@@ -50,8 +65,39 @@ cd platforms/ios && pod install
 
 and then re build the application.
 
-==== Development and Contributing
-Looking to use the showcase for development purposes with the SDK please take some time to read the link:./CONTRIBUTING.md[General Contributing Guide]
+===== Work with Self-signed Certificate
+
+By default, the app will not work with self-signed certificate due to security reasons. However, to help with local development, you may need to support it.  Here are the steps you can follow:
+
+1. Get the CA certificate of the server. You can use this command:
++
+[source, bash]
+----
+openssl s_client -showcerts -connect host:port
+---- 
++
+It will print out the full certificate chain of the server and you should save the content of the root certificate into a file.
+
+2. Drag and drop the .crt file onto the emulator/simulator you are using.
+
+3. On Android, double click on the .crt file and follow the instructions to install.
+
+4. On iOS, follow the on screen prompts to install the cert. Then navigate to Settings > General > About > Certificate Trust Settings and enable full trust for the cert.
+
+==== Contributing
+
+* link:./CONTRIBUTING.md[General Contributing Guide]
+* link:./docs/contrib/contributing-guide.adoc[Service Contributor Guide]
```

Comment:
Can we change this links to the full link to avoid any dead links over on the docs.